packages/eslint-plugin-kolibri/lib/utils.js

```diff
@@ -211,4 +211,33 @@ module.exports = {
       });
     });
   },
+
+  /**
+   * Report unused translation definitions.
+   */
+  reportUnusedTranslations(context, definitions, uses) {
+    const unused = definitions.filter(prop => !uses.includes(prop.name));
+
+    unused.forEach(prop => {
+      context.report({
+        node: prop.node,
+        message: `Unused translation defined for ${prop.node.name}`,
+      });
+    });
+  },
+
+  /**
+   * Report uses of undefined strings
+   */
+  reportUseOfUndefinedTranslation(context, definitions, uses) {
+    const definedStrings = definitions.map(prop => prop.name);
+    const badAttempts = uses.filter(prop => !definedStrings.includes(prop.value));
+
+    badAttempts.forEach(node => {
+      context.report({
+        node,
+        message: `Attempt to use undefined string ${node.value}`,
```

Comment:
Maybe just something more matter-of-fact like `"<string_id>" is not defined in the $trs option`